docs/index.rst

```diff
@@ -3,20 +3,34 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Liquidprompt's documentation!
-========================================
+Welcome to the Liquid Prompt's documentation!
+=============================================
```

Comment:
This may be discussed, as I feel that it emphasizes the adjective. I mentioned this idea in PR #677, if you feel it's legit. I will of course not fight about it :)